packages/Confirmation/src/Confirmation.js

```diff
@@ -73,15 +73,11 @@ const Confirmation = props => {
     popoverKey = uuid();
   }, [isPending]);
 
-  const triggerProps = {
-    isConfirmOpen,
-    handleOpenConfirm,
-  };
-
   const handleCloseConfirm = () => {
     if (isConfirmOpen) {
       setIsConfirmOpen(false);
-      if (triggerRef.current) triggerRef.current.focus();
+      // being called repeatedly by popoverclose and cancel button?
+      console.log("calling close confirm");
```

Comment:
@nahumzs Any idea why this would be getting called repeatedly when clicking the cancel button in confirmation popup?
![image](https://user-images.githubusercontent.com/4040102/70955874-1ff22e00-2027-11ea-9ffc-51b7c0710a68.png)